check-pr-status.js

```diff
@@ -0,0 +1,30 @@
+// fetch /repos/{owner}/{repo}/pulls/{pull_number}
+// and check its mergeable_state
+// if "clean", exit with status code 0
+// else exit with error
+const { setTimeout } = require('timers/promises');
+
+(async () => {
```

Comment:
again, should use TLA (but maybe we don't need this separate script at all?)

Reply:
Would using the .mjs extension do the trick or should I create a package.json with type: "module"?
I used the .mjs extension but I can change the approach if you want.

Reply:
`.mjs` is reasonable IMO 👍